src/Framework/Microsoft.Internal.Aspnetcore.DotNetApiDocs.Transport/src/Microsoft.Internal.Aspnetcore.DotNetApiDocs.Transport.csproj

```
@@ -0,0 +1,77 @@
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <IsShippingPackage>false</IsShippingPackage>
+    <IsPackable>true</IsPackable>
+    <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <IncludeSymbols>false</IncludeSymbols>
+    <NoPackageAnalysis>true</NoPackageAnalysis>
+    <RefAssemblyPackagePath>ref/$(TargetFramework)/</RefAssemblyPackagePath>
+    <RequiresDelayedBuild>true</RequiresDelayedBuild>
+    <IsProjectReferenceProvider>false</IsProjectReferenceProvider>
+    <BuildHelixPayload>false</BuildHelixPayload>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="@(AspNetCoreShippingAssembly)" />
+
+    <!-- This project does not produce a .dll -->
+    <Reference Remove="Microsoft.Extensions.ApiDescription.Server" />
+  </ItemGroup>
+
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <EnablePackageValidation>false</EnablePackageValidation>
+  </PropertyGroup>
```

```suggestion
  <PropertyGroup>
    <EnablePackageValidation>false</EnablePackageValidation>
  </PropertyGroup>
```

nit: move this up to the other properties

---

I put it here (below SDK.targets import) because the setting in D.B.T is unconditional: https://github.com/dotnet/aspnetcore/blob/595abfb73d77a7c6d16485fe2e8bd5133ef209fc/Directory.Build.targets#L159. Probably I should just condition that property on the property not being empty. I'll take this opportunity to do that.